jabgui/src/main/java/org/jabref/gui/preferences/JabRefGuiPreferences.java

```diff
@@ -424,6 +429,7 @@ public EntryEditorPreferences getEntryEditorPreferences() {
                 getBoolean(AUTOLINK_FILES_ENABLED),
                 EntryEditorPreferences.JournalPopupEnabled.fromString(get(JOURNAL_POPUP)),
                 getBoolean(SHOW_SCITE_TAB),
+                getBoolean(SHOW_CITATION_CONTEXT_TAB),
```

Comment:
There already have been made changes here, so you will need to fix merge conflicts